modules/monitoring-applying-custom-alertmanager-configuration.adoc

```diff
@@ -67,15 +76,15 @@ route:
   repeat_interval: 12h
   receiver: default
   routes:
-  - match:
-      alertname: Watchdog
+  - matchers:
+    - alertname: Watchdog
```

Comment:
error in alertmanager
```
ts=2023-02-06T02:39:34.777Z caller=coordinator.go:118 level=error component=configuration msg="Loading configuration file failed" file=/etc/alertmanager/config_out/alertmanager.env.yaml err="yaml: unmarshal errors:\n  line 10: cannot unmarshal !!map into string\n  line 14: cannot unmarshal !!map into string\n  line 17: cannot unmarshal !!map into string"
```
should be
`- "alertname=watchdog"`

Reply:
fixed